logsearch-jobs.yml

```
@@ -105,6 +109,12 @@ instance_groups:
         app_index_settings:
           index.mapping.total_fields.limit: 2000
           index.queries.cache.enabled: "false"
+        base_index_component_name: c-index-mappings-lfc
+        app_index_component_name: c-index-mappings-app-lfc
+        platform_index_component_name: c-index-mappings-platform-lfc
```

Updating the names to match the changes above:

```suggestion
        base_index_component_name: index-mappings-lfc
        app_index_component_name: index-mappings-app-lfc
        platform_index_component_name: index-mappings-platform-lfc
```